docs/application/tizen-studio/rt-ide/getting-started/debug.md

```diff
@@ -28,23 +28,59 @@ To debug the application:
 
    ![Debugging the project](./media/rt_debug.png)
 
-4. In the debug config select dialog, select a debug flashing option and click **OK**.
+4. In the **Select GDB Client Binary** dialog, select the path where arm gdb is installed and click **OK**. 
+	> **Note**  
+	> gdb installation path can be found using the command **which arm-none-eabi-gdb** in terminal. 
 
    ![Debug configuration](./media/rt_debug_config.png)
 
-5. Before the debugging starts, the Confirm Perspective Switch dialog is shown.
-
-   ![Perspective switch](./media/rt_debug_switch.png)
-
-   To switch to the **Debug** perspective, click **Yes**.
-
-   The **Debug** perspective shows the views useful for debugging activities, organized in the optimal fashion within the Tizen Studio window.
-
-   ![Debug perspective](./media/rt_debug_perspective.png)
-
-   When an action is executed on the board and it passes through a break point, the line is in the suspended state (see the figure above).
-
-   You can perform various debugging functions (such as resume, step into, and step over) by using the toolbar in the **Debug** perspective.
+5. Before the debugging starts, the Confirm Perspective Switch dialog is shown. Click **No** and add **Source Mapping** before proceeding to debug perspective.
+
+   ![Debug perspectiveno](./media/rt_debug_confirmno.png)
+   
+6. To add **Source Mapping**, click **Debug Configurations** in the debug dropdown.
+
+   ![Debug configurations](./media/rt_debug_config_menu.png)
+
+7. Expand the **GDB OpenOCD Debugging** from the list in the left and select the available configuration. Select the **Source** tab and click **Add** button.
+
+	![Debug sourceadd](./media/rt_debug_source.png)
+   
+8. Select **Path Mapping** option from the **Add Source** dialog.
+
+	![Debug sourcepath](./media/rt_debug_pathadd.png)
+	
+9. In the **Path Mapping** dialog click **Add** button and enter the name for new path mapping.
```

Comment:
In the **Path Mapping** dialog box, click **Add** , and enter the name for new path mapping.